src/helpers/graph.js

```diff
@@ -0,0 +1,42 @@
+export function removeNoteFromGraph(nodes, edges, noteToDelete) {
+  let newNodes = [ ...nodes ]
+  let newEdges = [ ...edges ]
+  let parentEdge = newEdges.find(edge => edge.to === noteToDelete.id)
+
+  removeChild(noteToDelete.id)
+  newNodes = newEdges.find(edge => edge.from === parentEdge.from) ? //check if there are siblings of deleted node if not revoke parant status
+             newNodes : newNodes.map(node => { return node.id === parentEdge.from ? { ...node, group: 'children' } : node })
+  return { notes: newNodes, edges: newEdges }
+
+  function removeChild(childNoteId) {

```

Comment:
When you call this function you pass `noteToDelete.id`. But the function expects `childNoteId`.

So is it `noteToDelete.id` or `childNoteId` ? A note to delete and a child note are a bit different things I think.

Also, in this file you operate with `nodes` but pass `noteToDelete` and `childNoteId`.
Let's use `noDe` word everywhere, since it's a graph utility and it knows nothing about `noTes`?